instrumentation/opentelemetry-instrumentation-botocore/tests/bedrock_utils.py

```diff
@@ -275,8 +275,6 @@ def remove_none_values(body):
             continue
         if isinstance(value, dict):
             result[key] = remove_none_values(value)
-        elif isinstance(value, list):
```

Comment:
This does not make any sense since the function assumes `value` is a mapping with the `items` method